### ISSUE TITLE
Update for latest virtual-dom

### ISSUE THREAD

**Issue by WorldMaker** [Swatinem/jade-virtualdom#8]
It appears that this has gotten out of sync with recent changes to virtual-dom: this is producing `tag` keys, but virtual-dom seems to want `tagName` now. Perhaps this should instead compile to use the `virtual-dom/h` virtual hyperscript helper to maybe avoid future object representation changes?

**Comment by Swatinem** [comment]
This was never intended for the npm `virtual-dom` anyway, but rather https://github.com/Swatinem/virtualdom

The experience I have made with `virtual-dom/h` is that it is horribly slow, sometimes spending as much as ⅓rd of the time in there. (I have used mercury in another project)

Anyway, I am not really maintaining this any more. I am happy to add you as a maintainer if you are willing :-)

**Comment by WorldMaker** [comment]
Yeah, I wanted a template language for [Cycle](http://cycle.js.org), which uses `virtual-dom`, and already had a node app that was using Jade server-side... I'll take a look and see if I can come up with a decent approach to support `virtual-dom`.

**Comment by Swatinem** [comment]
I added you as a maintainer so knock yourself out :-)
